### PR TITLE
My room engagement dialogues

### PR DESCRIPTION
**Pull request by contingency40**
All of the My Room engagement dialogues (the gifting of the ring) should now be done or checked. All of the My Room files that don't have additional EX or DLC dialogue should also be 100% complete with no MTLed.

### CODE REVIEW

File: Gaijinizer patch/Map024.txt [hunk 1]
```diff
@@ -482,8 +482,8 @@ been engaged?」#MTLed
 　\\C[2]人間界を移動できる彼女たちに対して、そんな事して…」
 > CONTEXT: Map024/events/42/pages/1/5/Dialogue
 > CONTEXT: Map194/events/42/pages/1/5/Dialogue
-Cat「I do not know ...... \\C[2] So for those who can move the
-human realm ...」#MTLed
+Cat「They won't know... \\C[2] So for those who can move the
+human realm ...」#ConfusedTranslatorNoises
```
Comment (Strate-S) [inline Gaijinizer patch/Map024.txt:486]: Maybe this is supposed to be something along the lines of "to do such a thing (cheat/get engaged multiple times) to a group of girls who can move between human worlds (i.e. are aware of what happens in other timelines and can see you cheating on them)", with the implication being that it is an extremely stupid idea?
Just a thought that crossed my mind, still not sure.

File: Gaijinizer patch/Map062.txt [hunk 2]
```diff
@@ -1007,17 +1010,20 @@ Are you sure ...?#MTLed
...
  ママは、ボウヤを残してどこにも行ったりはしないわ。
 > CONTEXT: Map062/events/5/pages/2/19/Dialogue/enemy4
-.........................................
-............ even if there is nothing like that
-Mom won't go anywhere with the boy left.#MTLed
+............
+Mommy would never go off somewhere and leave her
+little boy behind, even if you hadn't given her
+something so amazing.#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map062.txt:1017]: Note

File: Gaijinizer patch/Map062.txt [hunk 3]
```diff
@@ -1026,92 +1032,97 @@ Did you say that? I am such a demon ......#MTLed
...
+mommy and her little boy, that we'll be together
+forever...#Ron
 > END STRING
 
 > BEGIN STRING
 『ゆびきりげんまん』……ね？
 > CONTEXT: Map062/events/5/pages/2/60/Dialogue/enemy4
-"Yubikiri Genman"...right?#MTLed
+A "oinky promise", right?#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map062.txt:1096]: I assume this is a "pinky promise"?

File: Gaijinizer patch/Map070.txt [hunk 4]
```diff
@@ -2019,76 +2028,81 @@ Yup.sure.\\N[2]. I prepared for that.#MTLed
 \\N[1]くんが、ワタシの『永久貢ぎ主』に
  なってくれるなんて………♡
 > CONTEXT: Map070/events/5/pages/2/33/Dialogue/enemy5
-I'm glad...... ♡. \\N[1]-kun became 'I''s
-'Permanent tribute'. It will be...... ...#MTLed
+I'm so happy... ♡
+
+\\N[1]-kun's going to become my
+"permanent donor"...#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map070.txt:2035]: Note

File: Gaijinizer patch/Map080.txt [hunk 5]
```diff
@@ -919,71 +924,77 @@ I brought "this" only because I was not lost.#MTLed
...
 全てひっくるめて、君が大好きなんだ。
 > CONTEXT: Map080/events/5/pages/2/33/Dialogue/kazuya
-Of course, "Body" does not matter to your
-attraction...I am not going to say anything
-beautiful. It's all wrapped up, I love you.#MTLed
+I'm not going to bullshit you with lines about
+how this has nothing to do with how attractive
+your "body" is. This is about how much I love
+every aspect of you.#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map080.txt:931]: Note

File: Gaijinizer patch/Map085.txt [hunk 6]
```diff
@@ -51,13 +51,13 @@ I don't really have a use for that...#Ron
 もうちょっと女の子が喜ぶモノにしてよー？
 > CONTEXT: Map085/events/5/pages/0/203/Dialogue/enemy6
 Being a child means "Det her off" by the demon
-standard. Have you made the girl happy again?#MTLed
+standard. Have you made the girl happy again?#CheckLater
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map085.txt:55]: Note

File: Gaijinizer patch/Map085.txt [hunk 7]
```diff
@@ -1173,24 +1183,26 @@ But stop the idol......... come to this room
...
  『大きくなったら、アイドルになる』って
 > CONTEXT: Map085/events/5/pages/2/43/Dialogue/enemy6
-I don't know my face, but...maybe a father-like
-person. A girl like me is saying:
-"If you grow up, you become an idol"#MTLed
+I'm not sure who they were, but I remember a
+little girl like me saying to someone who was
+probably like her father, "I want to be an idol
+when I grow up"...#RonIffyFlow
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map085.txt:1190]: Note

File: Gaijinizer patch/Map122.txt [hunk 8]
```diff
@@ -1109,51 +1109,57 @@ like this.#MTLed
...
  蛇悪魔とか、性悪とか
 
  そういうのをひっくるめた君の返事が欲しいんだ。
 > CONTEXT: Map122/events/5/pages/2/25/Dialogue/kazuya
-Snake demon or evil. I want your reply to wrap up
-that.#MTLed
+All that stuff about being a snake demon and
+having a bad personality, I just want to hear
+your answer...#RonLocalizationIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map122.txt:1136]: Note

File: Gaijinizer patch/Map122.txt [hunk 9]
```diff
@@ -1109,51 +1109,57 @@ like this.#MTLed
...
 
 > BEGIN STRING
 欲求不満だとか、そんな単純な理由じゃない…
 > CONTEXT: Map122/events/5/pages/2/29/Dialogue/kazuya
-It's not frustration, it's not such a simple
-reason ...#MTLed
+It's nothing so simple as sexual frustration...#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map122.txt:1142]: "so" -> "as"

File: Gaijinizer patch/Map070.txt [hunk 10]
```diff
@@ -2019,76 +2028,81 @@ Yup.sure.\\N[2]. I prepared for that.#MTLed
...
 > BEGIN STRING
 …………けど、分かってるね？
 
 ハルの『永久貢ぎ主』になるって意味は。
 > CONTEXT: Map070/events/5/pages/2/76/Dialogue/enemy7
-...... But you understand? What does it mean to
-be Haru's "Permanent tribute"?#MTLed
+But... do you understand what it means to be
+Haru's "permanent donor"...?#Ron
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map070.txt:2098]: same as the other comment in this file.

File: Gaijinizer patch/Map122.txt [hunk 11]
```diff
@@ -1080,27 +1080,27 @@ has reached 100!#Triketos
...
 
 人間の一世一代の『プロポーズ』をかますなんて……
 > CONTEXT: Map122/events/5/pages/2/6/Dialogue/enemy14
-Anything like this demon ...... and a wicked man
-What a man's first generation "propose" is to be
-destroyed ...#MTLed
+Trying to surprise a demon like me, much less
+one with a horrible personality with your human
+once-in-a-lifetime marriage proposal...?#RonIffyWording
 > END STRING
```
Comment (Triketos) [inline Gaijinizer patch/Map122.txt:1094]: Think this is a bit better:
>_**Are you trying to surprise me, a demon
with a horrible personality, with your "human"
once-in-a-lifetime marriage proposal...?**_

File: Gaijinizer patch/Map024.txt (hunk 1 above)
Comment (Triketos) [reply Gaijinizer patch/Map024.txt:486]: We had a good chat on this. Basically the cat is talking to _you_, the player, breaking the fourth wall a bit. We decided on this:
```
Cat「This guy won't notice, \\C[2] but the girls who can
move across human timelines might pick up on what
you're doing...」
```

**Review by Triketos — APPROVED** [review]

**Pull request merged**